views/login.php

```diff
@@ -41,8 +41,6 @@
         $_SESSION['password'] = $loginQuery;
         $_SESSION["errorMessage"] = "Invalid login or password";
         header("Location: /Login");
-        $_SESSION['user'] = $email;
-        header("Location: /");
```

Comment:
wydaje mi się że zostawiłeś po swoim commicie tego header do location:/ i przez to nawet jak nic nie wpisywało się w login albo hasło to traktowało ciebie jako zalogowanego usera

Reply:
My bad fam